data/reusables/dependabot/supported-package-managers.md

```diff
@@ -116,9 +117,11 @@ For {% data variables.product.prodname_dependabot_security_updates %}, Gradle su
 
 {% ifversion dependabot-PEP621-support %}
 
-#### pip and pip-compile
+#### pip-compile and uv
 
-In addition to supporting updates to `requirements.txt` files, {% data variables.product.prodname_dependabot %} supports updates to `pyproject.toml` files if they follow the PEP 621 standard.
+{% data variables.product.prodname_dependabot %} can update `requirements.txt` compiled files using both `pip-compile` and `uv` (depending on which one was used to generate the file), as well as performing updates to `pyproject.toml` files if they follow the PEP 621 standard.
```

Comment:
I think I would clarify, `uv pip` here to distinguish from the top-level uv interface.